pytorch_blade/tests/torchscript/shape_analysis_tool.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The BladeDISC Authors. All rights reserved.
```

Comment:
This filename is too general, maybe `file_checker` is better? And pass the `shap_analysis_pass` as argument.

Reply:
This is not a file checker. It just read in graph, do shape analysis, and output the graph again. FileCheck is handler with llvm lit.